pymedphys/_gui/streamlit/mudensity-compare.py

```diff
@@ -814,33 +818,46 @@ def trf_input_method(patient_id="", key_namespace="", **_):
 
     headers
 
+    deliveries = cached_deliveries_loading(tables, delivery_from_trf)
+
+    individual_identifiers = [
```

Comment:
patient id ?  DICOM Patient ID, MOSAIQ PAT_ID1 ?
a comment here would be helpful

Reply:
Yup, these TRF files are indexed by MOSAIQ `Ident.PAT_ID1` during collection in our centre. That is what this Patient ID refers to.

See https://github.com/pymedphys/pymedphys/blob/f4a5efd41d0f48907686d3ee562ec1c3bf42fadb/pymedphys/_mosaiq/delivery.py#L122-L123

Will add comment.

Reply:
Just so it's easier to find here is a link to the added docstring:

https://github.com/pymedphys/pymedphys/blob/651d7aa7d7c4b25f884ce01bb32f80424a9a03eb/pymedphys/_gui/streamlit/mudensity/_trf.py#L70-L86